src/Settings/components/transliteration.jsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { useState } from "react";
+import { Text } from "react-native";
+import { ListItem, BottomSheet, Avatar, Icon, Switch } from "@rneui/themed";
+import STRINGS from "../../common/localization";
+import {
+  setTransliteration,
+  toggleTransliteration,
+  TRANSLITERATION_LANGUAGES,
+} from "../../common/actions";
+import { useSelector } from "react-redux";
```

Comment:
As mentioned above too, move third party imports before the internal imports

Reply:
Fixed